### PR TITLE
[Feat]: alert & button component added

### PR DESCRIPTION
**Pull request by t007rushi**

### CODE REVIEW

File: components/alert.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
+:root {
```
Comment (JantuDeb) [inline components/alert.css]: You can placed all these css variable in main css file,  don't need to use another :root for that.

**Review by JantuDeb — APPROVED** [review]
This looks great.

**Comment by hussiantabha** [comment]
Looks good and clean

**Comment by fahd23** [comment]
Looks good to me, One suggestion have one main CSS file and import all other components in that file

**Review by fahd23 — APPROVED** [review]
Looks good to me, One suggestion change what jantu suggested, That makes our work easy

**Review by Priya-Patel31 — APPROVED** [review]
Code looks clean , go ahead merge it !!!

File: components/alert.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  --sec-suc: rgb(195, 243, 215);
+  /* info */
+  --prim-info: rgb(113, 201, 255);
+  --sec-info: rgb(215, 240, 255);
+  /* warning */
+  --prim-war: rgb(255, 165, 3);
+  --sec-war: rgb(255, 219, 155);
+  /* danger */
+  --prim-dang: rgb(255, 72, 88);
+  --sec-dang: rgb(255, 224, 227);
```
Comment (horsemaker) [inline components/alert.css]: suc, war, dang may sound confusing 👀

File: components/alert.css [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+}
+.container {
+  background-color: var(--alert-primary-color);
+  max-width: 50%;
+  max-height: 100%;
+  margin: 2rem;
+  padding: 2rem;
+}
+
+.alert_wrap {
```
Comment (horsemaker) [inline components/alert.css]: shouldn't it be alert-wrap?

**Review by horsemaker — APPROVED** [review]

File: components/alert.css (hunk 3 above)
Comment (t007rushi) [reply components/alert.css]: Thank You will do that

File: components/alert.css (hunk 2 above)
Comment (t007rushi) [reply components/alert.css]: Yep made it readable

File: components/alert.css (hunk 1 above)
Comment (t007rushi) [reply components/alert.css]: Yep

**Pull request merged**